src/Compilers/Test/Core/Compilation/CompilationExtensions.cs

```diff
@@ -371,7 +371,7 @@ void checkControlFlowGraph(IOperation root, ISymbol associatedSymbol)
 
                     case IParameterInitializerOperation parameterInitializerOperation:
                         // https://github.com/dotnet/roslyn/issues/27594 tracks adding support for getting ControlFlowGraph for parameter initializers for local functions.
```

Comment:
>https://github.com/dotnet/roslyn/issues/27594 tracks adding support for getting ControlFlowGraph for parameter initializers for local functions. [](http://example.com/codeflow?start=27&length=144)

Should we adjust the comment to mention lambdas too? Should we mention lambdas in the referenced issue?

Reply:
Good catch; updated the comment (d0b06f2e19bfff645285321e72b12266f546ac92) and the issue.